docs/reference/transform/checkpoints.asciidoc

```diff
@@ -55,6 +55,47 @@ TIP: If the cluster experiences unsuitable performance degradation due to the
 {transform}, stop the {transform} and refer to <<transform-performance>>.
 
 
+[discrete]
+[[sync-field]]
+== Using the ingest timestamp for syncing the {transform}
+
+In most cases, it is strongly recommended to use the ingest timestamp of the 
+source indices for syncing the {transform}. This is the most optimal way for 
+{transforms} to be able to identify new changes. It can be set by using an 
+ingest pipeline. You can create an ingest pipeline either using the 
+<<put-pipeline-api, ingest pipeline API>> (like the example below) or via {kib} 
+under **Stack Management > Ingest Pipelines**. Use a `set` processor to set a 
+field – in the example, this is `_source.@timestamp_ingest` – and associate it 
+with the value of the ingest timestamp.
+
+[source,console]
+----------------------------------
+PUT _ingest/pipeline/set_ingest_time
+{
+  "description": "Set ingest timestamp.",
+  "processors": [
+    {
+      "set": {
+        "field": "_source.@timestamp_ingest",
+        "value": "{{{_ingest.timestamp}}}"
+      }
+    }
+  ]
+}
+----------------------------------
+
+After you created the ingest pipeline, apply it to the source indices of your 
+{transform}. The pipeline adds a field `_source.@timestamp_ingest` to every 
+document with the value of the ingest timestamp. Configure the 
+`sync`.`time`.`field` property of your {transform} to use the field by using the 
+<<put-transform>> for new {transforms} or the <<update-transform>> for existing 
```

Comment:
The automated title for these links doesn't contain the word "API", so I recommend specifying link titles:
```suggestion
<<put-transform,create {transform} API>> for new {transforms} or the <<update-transform, update {transform} API>> for existing 
```